views/shell.erb

```diff
@@ -1,32 +1,31 @@
 <div id="shell">
+<% if @current %>
   <iframe id="console" src="/shell/login/"></iframe>
+<% else %>
+  <h3>Puppet Education</h3>
   <p>
-    Shell accounts have been created for each student for editing code and
-    running Puppet interactively. Log in using the user account and credentials
-    you created.
+    Welcome to the Puppet classroom.  Please create a user account; creating a user will provision a Docker instance that will allow you to complete labs and exercies.  The Docker instance can be accessed through a web terminal once it has been created.
   </p>
-  <p><a href="#" id="show-alternate">Alternate Login Methods</a></p>
-  <div id="alternate" title="Alternate Login Methods">
-    <p>
-      If you prefer, you may also use any standard SSH client using the username
-      and password you specified. Some common SSH clients include:
-    <p>
-    <ul>
-      <li>
-        Built-in to Mac OSX and Linux:
-        <ol>
-          <li>Start Terminal.app or any terminal emulator to get to the command line</li>
-          <li>Run <code>ssh <%= request.host %></code></li>
-        </ol>
-      </li>
-      <li>
-        Windows:
-        <ul>
-          <li><a href="http://the.earth.li/~sgtatham/putty/latest/x86/putty.exe">PuTTY</a></li>
-          <li><a href="http://winscp.net/eng/download.php#download2" target="_new">WinSCP</a></li>
-          <li><a href="http://www.vandyke.com/download/securecrt/download.html" target="_new">SecureCRT</a></li>
-        </ul>
-      </li>
-    </ul>
-  </div>
+  <p>
+    If you prefer, you may also use any standard SSH client using the username and password you specified. Some common SSH clients include:
+  </p>
+  <ul>
+    <li>
+      Mac OSX and Linux:
+      <ul>
+        <li>Terminal.app</li>
```

Comment:
These are not SSH clients. The original wording was more accurate, though adding links or descriptions of the terminals is probably a good idea.